scala/build/maintenance.Dockerfile

```
@@ -256,6 +256,13 @@
 RUN pipx install awscli
 # RUN ln -s /home/qa/.local/bin/aws /usr/local/bin/aws
 
+# trcli
+RUN apt-get update && apt-get install -y python3-full python3-pip
```

Hadolint

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)